installer/manifests/keptn/charts/control-plane/templates/core.yaml

```diff
@@ -356,7 +356,13 @@ spec:
                   name: mongodb-credentials
                   key: password
             - name: MONGO_DB_NAME
-              value: 'keptn'
+              value: {{ .Values.mongodb.dbName | default "keptn"}}
```

Comment:
Do you really need the default here, if the name is already provided in the values file directly?

Reply:
the name should be provided in the values.yaml file, I just added the default to be safe